app/lib/frontend/templates/views/shared/search_banner.dart

```diff
@@ -16,17 +20,29 @@ d.Node searchBannerNode({
 }) {
   return d.form(
     classes: ['search-bar', 'banner-item'],
+    attributes: {
+      'autocomplete': 'off',
+    },
     action: formUrl,
     children: [
       d.input(
         classes: ['input'],
+        type: 'search',
         name: 'q',
         placeholder: placeholder,
-        autocomplete: 'on',
+        autocomplete: 'off',
         autofocus: autofocus,
         value: queryText,
         attributes: {
           'title': 'Search',
+          if (requestContext.experimentalFlags.isSearchCompletionEnabled)
+            'data-widget': 'completion',
+          'data-completion-base64': _initialCompletionData,
+          'data-completion-src': '/api/search-input-completion-data',
+          'data-completion-class': 'search-completer',
+          'data-completion-option-class': 'search-completer-option',
```

Comment:
Not sure: what is the role of these class names - do we need them as dynamic options?

Reply:
I think it's class for the dropdown, and the options. But I could be sold with us making the option and selected options hardcoded classes.

There is an attempt here to keep the completer generic enough that it could be applied to other things as well. Maybe that's overkill :see_no_evil:

Reply:
I think we can start with hardcoded now and add customization later - when needed?

Reply:
Okay, I'll simplify a bit..